fastlane/Fastfile

```diff
@@ -291,6 +292,12 @@ private_lane :appstore_api_key do
   )
 end
 
+lane :pod_lint do
+  sdk_names.each do |sdk_name|
+    pod_lib_lint(podspec: "#{sdk_name}.podspec", allow_warnings: true)
```

Comment:
Will this run `pod repo update` before? 🤔 At least locally, I've seen StreamChatUI.podspec failing linting because it says it does not the most updated cocoapods trunk. Lets see if this is needed or not 👍